README.md

```diff
@@ -434,3 +520,31 @@ Not all controls in the CIS Benchmark are capable of automated assessment. The t
 | 9.1.9     | Manual            |
 
 For any controls marked as 'Manual', please refer to the following following at [SAF-CLI](https://saf-cli.mitre.org/) on how to apply manual attestations to the output of an automated assessment. The following [link](https://vmware.github.io/dod-compliance-and-automation/docs/automation-tools/safcli/) that references the SAF-CLI is also useful.
+
+[top](#table-of-contents)
+
+## Authors
+[Center for Internet Security (CIS)](https://www.cisecurity.org/)
+ 
+[MITRE Security Automation Framework Team](https://saf.mitre.org)
+ 
+## NOTICE
+ 
+© 2018-2025 The MITRE Corporation.
+ 
+Approved for Public Release; Distribution Unlimited. Case Number 18-3678.
+ 
+## NOTICE
+ 
+MITRE hereby grants express written permission to use, reproduce, distribute, modify, and otherwise leverage this software to the extent permitted by the licensed terms provided in the LICENSE.md file included with this project.
+ 
+## NOTICE  
+ 
+This software was produced for the U. S. Government under Contract Number HHSM-500-2012-00008I, and is subject to Federal Acquisition Regulation Clause 52.227-14, Rights in Data-General.  
```

Comment:
We have a new NOTICE clause that doesn't have a specific HHS contract number on it. I'll swap this out.